### DISCUSSION TITLE
Jellyfin v4 Script - Hardware Acceleration error

### DISCUSSION THREAD

**Discussion by KaiserN** [tteck/Proxmox#762] [Help Needed]
**Describe the issue**
Il try to install Jellyfin, ended up with:
 - Setting Up Hardware Acceleration...‼ ERROR 100@92 Unknown failure occurred.

Host is up2date
Linux pveHLN1 5.15.74-1-pve #1 SMP PVE 5.15.74-1 (Mon, 14 Nov 2022 20:17:15 +0100) x86_64 GNU/Linux

**Screenshots**
![grafik](https://user-images.githubusercontent.com/47419452/203285501-218b8a1e-48a4-428e-9f00-b546bda3e6e9.png)

is there any logfile or something i can debug the issue? Please let me know, thx

**Comment by tteck** [comment]
Many factors go into getting "Hardware Acceleration" working correctly. Machines are different.
Things to consider, always run scripts via the Proxmox Shell (no SSH)
Try without "Hardware Acceleration" by creating Jellyfin unprivileged

**Comment by KaiserN** [comment]
Hi, so i tryed via Proxmox Shell, same issue

`
       __________    ____  _____________   __
      / / ____/ /   / /\ \/ / ____/  _/ | / /
 __  / / __/ / /   / /  \  / /_   / //  |/ / 
/ /_/ / /___/ /___/ /_v4/ / __/ _/ // /|  /  
\____/_____/_____/_____/_/_/   /___/_/ |_/   

Using Advanced Settings
Using Ubuntu Version: 22.04
Using Container Type: 0
Using Root Password: Automatic Login
Using Container ID: 107
Using Hostname: jellyfin
Using Disk Size: 8
Allocated Cores: 2
Allocated RAM: 2048
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Using MAC Address: Default
Using Vlan: Default
Creating a Jellyfin LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using zstore for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 107 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.200.228 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.121.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 - Setting Up Hardware Acceleration...‼ ERROR 100@92 Unknown failure occurred.`

Any hint for me?

kind regards

**Comment by tteck** [comment]
Beings I don't know you machine specs, try creating Jellyfin unprivileged

**Comment by KaiserN** [comment]
Host HW: Lenovo M900 Tiny, i5-6400T

looks like a version problem, with 20.4 default settings it turns on the HW acceleration
but then it stuck here 
![grafik](https://user-images.githubusercontent.com/47419452/203415393-d90b1a80-acf6-4494-bc2b-7666a0d99645.png)

maybe a slow mirror but idk
will try again tomorrow or install it manually, will see

E: btw. unprivileged 22.04 also failed, unprivileged 20.04 didnt test yet

regards

**Comment by TakeusV1** [comment]
Hi, the installation went well, but the acceleration does not work when launching a video (this client is not compatible with the media and the server does not send a compatible format), my processor is an i5-6500T, I tested with QSV and VAAPI, same...

**Reply by TakeusV1** [reply]
Good news, I put the path of ffmpeg, and it seems to work. 
https://github.com/tteck/Proxmox/issues/533

**Reply by KaiserN** [reply]
so you put this path `FFmpeg path: /usr/lib/jellyfin-ffmpeg/ffmpeg ` in the settings?
how did you test it?

the group isnt wright i guess?

Jellyfin LXC
```
pct exec 105 -- ls -l /dev/dri
total 0
drw-rw---- 2 root root         80 Nov 22 19:30 by-path
crw-rw---- 1 root video  226,   0 Nov 22 19:30 card0
crw-rw---- 1 root syslog 226, 128 Nov 22 19:30 renderD128
```

Host: 
```
crw-rw----  1 root video  226,   0 Nov 22 20:30 card0
crw-rw----  1 root render 226, 128 Nov 22 20:30 renderD128
```

**Reply by TakeusV1** [reply]
I haven't touched the groups, to test, I launch a video that requires transcoding, if it works it's good, if not, I have an error that prevents playback...

**Reply by johndoe0815** [reply]
I have tested Jellyfin yesterday. Installation took VERY long but worked. 
But: No matter which video I tried to play, received the same error, in browser and on iPhone app. HW transcoding seems to be even deactivated by default in the settings. 
So known issue and I need to adjust the ffmpeg path?
Saw reports on Reddit with the same issue but no solution, people going back to Plex (no problem there).

**Comment by KaiserN** [comment]
Default installation went well, did take some time. But its up and running. HW ACC didnt test yet.

But if u try to install it with ubuntu 22.04 it fails all the time. Wich version did you use?

**Reply by TakeusV1** [reply]
Yes me too it didn't work with ubuntu 22.04, with ubuntu 20.04 it works... (with 22.04 same error as you at first at installation)

**Reply by KaiserN** [reply]
@tteck fyi
